src/data/causes/seas.json

```diff
@@ -0,0 +1,55 @@
+{
+  "path": "seas2",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "#5094FB",
+        "secondary": "#29BEBA",
+        "primaryContrast": "#fff"
+      }
+    },
+    "causeId": "SGa6zohkY",
+    "metadata": {
+      "title": "Tab for TeamSeas - Home",
+      "ogTitle": "Remove trash from the seas for free | Tab for #TeamSeas",
+      "url": "/seas/",
+      "ogDescription": "Open browser tabs to clean our seas, for free. It's that easy. Join #teamseas today!",
+      "ogImage": "../../img/cats/1.png",
+      "causeSpecificKeywords": "charity, seas, TeamSeas, teamseas, #teamseas, ocean, oceans, sharks, whales, dolphins, fish, environment, global warming, river, rivers, lake, lakes, pollution, extension, new tab, chrome, help, donation, raise money, money, easy, ways to donate, free, best, home, youtube, safe, volunteer, internet, tab for a cause, impact, legitimate, food, facebook, twitter, reddit, instagram, tumblr"
+    },
+    "sections": {
+      "landing": {
+        "title": "The easiest way to save our seas",
+        "subtitle": "Every tab opened raises money for #TeamSeas, an internet-led movement raising $30 million to clean up 30 million pounds of trash from our oceans, rivers, and lakes.",
+        "ctaImg": "../../img/cats/1.png",
+        "waveColor": "#5094FB"
```

Comment:
![Screen Shot 2021-12-06 at 9 33 40 AM](https://user-images.githubusercontent.com/77807117/144893819-12e09eda-bc69-47b1-bcea-5615bbe7cec1.png)

looks like the wave is missing.  Not exactly sure what's going on, but you'll need to add 'waveColor' to the graphql query in gatsby-node-esm as well.  Here's a tutorial on how the system is working for context: https://www.gatsbyjs.com/docs/tutorial/part-4/
but the tldr is to just specify 'waveColor' under ctaImg on line 157 of gatsby-node.esm.js